qiime2/sdk/result.py

```diff
@@ -237,10 +237,12 @@ def import_data(cls, type, view, view_type=None):
             format_ = view_type
 
         provenance_capture = archive.ImportProvenanceCapture(format_, md5sums)
-        return cls._from_view(type_, view, view_type, provenance_capture)
+        return cls._from_view(type_, view, view_type, provenance_capture,
+                              level='max')
 
     @classmethod
-    def _from_view(cls, type, view, view_type, provenance_capture):
+    def _from_view(cls, type, view, view_type, provenance_capture,
+                   level='min'):
```

Comment:
maybe call it `validate_level`, because we are pretty far removed from the code that does that, so out of context, `level` may not make very much sense.